Automattic/jetpack
# My Plan: update page for all paid plans

#### Changes proposed in this Pull Request:

* Similar to https://github.com/Automattic/jetpack/pull/11418
* Update My Plan page for paid plans:
  * Make plan feature cards more useful and actionable.
  * Reorder cards so that Security & Performance are the top two focuses.

#### Testing instructions:

* Create a [new Jetpack site using this link](https://jurassic.ninja/create?jetpack-beta&branch=update/my-plan-page-for-paid-plans&wp-debug-log).
* Connect Jetpack and buy a plan at the end of the flow.
* Go back to wp-admin and open the My Plan page.
* Make sure the page shows a focus on security and performance, and that all cards have a corresponding action.

#### Considerations:

- The number of blocks depends on the features available to different plans; please test this PR thoroughly.
- Having an odd number of features will make the last card span across the two columns. This was decided in https://github.com/Automattic/jetpack/pull/10911 but we can revisit that.
- I've decided to keep blue buttons for features that need activation; they will turn white and change copy once activated.
- We should create a new illo for SEO Tools; any ideas?

#### Before

<img width="1057" alt="screenshot 2019-03-06 at 17 01 33" src="https://user-images.githubusercontent.com/390760/53899184-89bdc000-4031-11e9-8abd-05a1b8b49178.png">

#### After

![image](https://user-images.githubusercontent.com/390760/54276874-48279a80-4586-11e9-989f-47c5e9e8e5ee.png)

#### Proposed changelog entry for your changes:

* My Plan page is now more useful and actionable for all plans.
